docs/src/administration/web/configure-project.md

```diff
@@ -9,10 +9,26 @@ description: |
 
 ## General
 
-From the first page of the project settings, `General`, you can update the project name, or navigate to other project settings options on the left side of the screen.
+From the first page of the project settings, `General`, you can update the project name, set the timezone for the project or navigate to other project settings options on the left side of the screen.
 
 ![configure project](/images/management-console/settings-basics.png "0.7")
 
+The project timezone affects:
+
+- Automated backups.
+- Maintenance windows.
+
+The project timezone doesn't affect:
+
+- App runtime.
+- System logs.
+- Crons and any subsequent tasks (like backups set through crons).
```

Comment:
The benefit of a list is the structure it adds to all the information. I wouldn't have a list with the items and then the details below, since then you lose the chance to skim for the one you want. The links should be in the list rather than below. That also allows you to get rid of the sentence `App runtime and cron timezone can be configured separately` as it doesn't add any additional information.

Logs should also have a link so people can know what timezone is in effect there.